README.md

```diff
@@ -25,6 +25,7 @@
     - [Semantic cache](#semantic-cache)
     - [Advanced RAG](#advanced-rag)
     - [Recommendation systems](#recommendation-systems)
+    - [Chat Session Management](#chat-session-management)
```

Comment:
Let's say `LLM Session Management` here instead of Chat? just so it's clear to a wider audience what we are covering here